7.1/Dockerfile

```diff
@@ -45,19 +46,24 @@ ARG PHP_EXTENSIONS="\
     bcmath \
     bz2 \
     sockets \
+    soap \
+    pcntl \
     opcache"
 
 
 ARG CFLAGS="-I/usr/src/php"
 RUN apk update \
     && apk add --no-cache \
-    shadow \
-    procps \
-    openssh \
-    curl \
     bash \
-    postgresql-client \
+    curl \
+    git \
```

Comment:
Why do we need git for?

Reply:
I checked the whole chain and actually, due to npm (and composer in some cases), we anyway need to install it in development and testing scenarios. As we already now have included here packages for convenience, which aren't necessary in minimal, production containers (bash, curl, netcat, ssh, mysql-client, postgres-client) - I decided to move git here as well. When we approach the topic of building smallest possible container, this repository (or branches) will be splitted.